start/scaling-up/submit-multiple-jobs.md

```diff
@@ -0,0 +1,286 @@
+[title]: - "Easily Submit Multiple Jobs"
+
+[TOC]
+
+# Easily Submit Multiple Jobs Using HTCondor
+
+## Overview
+
+HTCondor has several convenient features for streamlining high-throughput 
+job submission. This guide provides several examples 
+of how to leverage these features to **submit multiple jobs with a 
+single submit file.**
+
+*Why submit multiple jobs with a single submit file?*
+
+As described in our [Policies for using an OSG Connect submit server](https://support.opensciencegrid.org/support/solutions/articles/12000074852), 
+users should submit multiple jobs using a single submit file, or where applicable, as few 
+separate submit files as needed. Using HTCondor multi-job submission features is more 
+efficient for users and will help ensure reliable operation of the the login nodes.
+
+Many options exist for streamlining your submission of multiple jobs, 
+and this guide only covers a few examples of what is truly possible with 
+HTCondor. If you are interested in a particular approach that isn't described here, 
+please contact [OSG Connect support](mailto:chtc@cs.wisc.edu) and we will 
+work with you to identify options to meet the needs of your work.
+
+## Submit Multiple Jobs Using `queue`
+
+All HTCondor submit files require a `queue` attribute (which must also be 
+the last line of the submit file). By default, `queue` will submit one job, but 
+users can also configure the `queue` attribute to behave like a for loop 
+that will submit multiple jobs, with each job varying as predefined by the user.
+
+Below are different HTCondor submit file examples for submitting batches of multiple 
+jobs and, where applicable, how to indicate the differences between jobs in a batch 
+with user-defined variables. Additional examples and use cases are provided futher below:
+
+1.  ***[queue \<N\>](#process)*** - will submit *N* number of jobs. Examples 
+    include performing replications, where the same job must be repeated *N* number 
+    of times, looping through files named with numbers, and looping through 
+    a matrix where each job uses information from a specific row or column.
+2.  ***[queue \<var\> from \<list\>](#foreach)*** - will loop through a 
+    list of file names, parameters, etc. as defined in separate text file (i.e. *\<list\>*). 
+    This `queue` option is very flexible and provides users with many options for 
+    submitting multiple jobs.
+3.  **[Organizing Jobs Into Individual Directories](#initialdir)** -
+    another option that can be helpful in organizing multi-job submissions.
+
+These `queue` options are also described in the following video from HTCondor Week 2020: 
+![video thumbnail](https://raw.githubusercontent.com/OSGConnect/connectbook/master/images/multi-job-submit-video-thumbnail.png "2020 HTCondor Week Presentation") [Submitting Multiple Jobs Using HTCondor Video](https://www.youtube.com/watch?v=m7dQChJH5LU)
```

Comment:
I think these need to be nested somehow, like: 

`[![vid thumbnail](http://etc.)](youtubelink)`

Reply:
Do you mean for making the image a link to the video? I was just thinking to have the image and the link text separate, but I like your idea too, really just testing this out to see what it looks like, can modify later if once we have a better sense of how it looks and if we like it

Reply:
Ah! your idea makes sense too. Let's stick with what you have for now.